content/departments/product-engineering/product/process/user_research/user_research_candidates_from_data.md

```diff
@@ -0,0 +1,22 @@
+# User research candidates from anonymous data
+
+## Background
+
+Since we collect more data on Sourcegraph Cloud than self-hosted, this means we have the ability to segment users more granularly for user research. For example, we could retrieve the top 100 users of feature x, or get all users who performed a search but did not install the browser extension.
+
+## Principles
+
+A couple thoughts from teammates on what to think about when considering whether a segment of users is okay to reach out to:
+
+- If it's information that we can reasonably expect that our users will know we have access to, all good. If it's information that our users might not conceptualize we'd have access to, then we should reconsider.
+- Would any member of our team have trouble defending our practices in a public space? At a high level, what is being requested here seems fine to me (if we're not worried about cloud user emails being shared, and assuming the users have opted in to emails if necessary, etc.).
+
+## Legalities
+
+Legally, we can reach out to individual cloud users (but not cloud users within an organization) because individual users agree to our Privacy Policy when they sign up. Our Privacy Policy allows us to reach out to users for marketing purposes and product improvement purposes.
```

Comment:
> we can reach out to individual cloud users (but not cloud users within an organization)
I've learned that cloud is a bit of a loaded word here. For clarity when we're talking about individual cloud users, do we mean those who've created an account on sourcegraph.com?
Then when we talk about "not cloud users within an organization" what do we mean? Do we mean cloud users who've joined a cloud organization? Or do we mean people who've used sourcegraph.com and are also part of a paid on-prem organization?